auslib/db.py

```diff
@@ -805,107 +857,6 @@ def getChange(self, change_id=None, column_values=None, data_version=None, trans
             return None
         return changes[0]
 
-    def getPrevChange(self, change_id, row_primary_keys, transaction=None):
```

Comment:
Ah, I should've added a note about this. As far as I can tell, `rollbackChange` was never actually used. It was added in https://bugzilla.mozilla.org/show_bug.cgi?id=747129, but the current UI has always made a `POST` to `/releases/:release/revisions` to revert, which has always used `Releases.update` to do the work.

So, this is just long overdue dead code removal that I noticed while auditing what `GCSHistory` needs to implement.

As far as more changes to the UI goes, you're right - I need to update the revert controller/service to use a `POST` to `/releases/:release` instead of the `revisions` endpoint.

Reply:
Ah, that makes sense. There is the 'Revert to this' UI on the rules history to check on, not sure what that code that uses.

*gentle nudge about the coverage comment above*

Reply:
> There is the 'Revert to this' UI on the rules history to check on, not sure what that code that uses.

That uses a POST to `/api/rules/:rule_id/revisions` which eventually ends up doing a direct `update` at https://github.com/mozilla/balrog/blob/master/auslib/web/admin/views/history.py#L136

> gentle nudge about the coverage comment above

Thanks, I'll address the coverage in a separate PR.